ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-postsubmits.yaml

```diff
@@ -439,6 +439,79 @@ postsubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    cluster: build05
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+      timeout: 5h0m0s
+    labels:
+      ci-operator.openshift.io/cloud: aws
+      ci-operator.openshift.io/cloud-cluster-profile: aws
+      ci-operator.openshift.io/variant: ocp-4-10
+      ci.openshift.io/generator: prowgen
+    max_concurrency: 1
```

Comment:
We usually changes set `max_concurrency: 6` on postsubmit jobs

Reply:
this is generated config, though, right? Are we meant to update the concurrency?

Reply:
This is a generated file, created by `make jobs`. Is there some make target that I should run?

Reply:
I have rebased, but I haven't done anything about the concurrency.

Reply:
I have set max_concurrency to 6 in stackrox-stackrox-master-postsubmits.yaml

Reply:
> this is generated config, though, right? Are we meant to update the concurrency?

Sorry, Jouko and I were discussing this in DMs. fysa @RTann, here was my DM:
> I usually think of generated files like I do for grpc protobufs, sqlgen, and mockery, where the intention for the files those generate are to never be manually changed and everything you need from the generated files in configurable in the files you do interact with. However, for some reason or another, the OSCI folks decided not to treat the jobs the same way. The PR will fail if you change a generated value except for these: https://docs.ci.openshift.org/docs/how-tos/contributing-openshift-release/#tolerated-changes-to-generated-jobs
If you cross reference some of these keys with the reference YAML: https://steps.ci.openshift.org/ci-operator-reference
You'll notice that some aren't even configurable in the config, max_concurrency being one of them, unfortunately

Not trying to call out the OSCI maintainers. I'm sure there are good reason to have them separate, but yeah, even though they're generated files, we still need to edit some portion of them sometimes 🙂